frontend/packages/kubevirt-plugin/src/components/modals/add-template-source/add-template-source.tsx

```diff
@@ -29,40 +31,45 @@ const getAction = (dataSource: string): string => {
     case DataVolumeSourceType.HTTP:
     case DataVolumeSourceType.REGISTRY:
     case DataVolumeSourceType.S3:
-      return 'import';
+      return i18next.t('kubevirt-plugin~import');
```

Comment:
this text is used to construct a sentence. I think it will be hard to translate correctly so we should avoid doing so in the future. In my PR Im returning the whole sentence https://github.com/openshift/console/pull/7349/files#diff-b9e2d080ccc211c6940ba931b9cc4ced54956b21995895592f1517172738a2a7R34 I think we should use the same approach for other cases we have